index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>To-Dos App</title>
+</head>
+  <body>
+      <h1>To-Dos</h1>
+      <form action="#">
+        <input type="text" id="taskInput" placeholder="Enter a task">
+        <input type="submit" value="+">
+      </form>
+      <br>
+      <form id="checkboxForm">
+        <!-- Checkboxes will be added here -->
+        <button type="submit">Submit Response</button>
```

Comment:
@jjw18th Why is this a button of type submit? I see you have use an input of type submit before this. Please look look into the differences between an input of type submit, a button of type submit and a plain button and determine which one is appropriate for our use cases.

Reply:
Tracked by #3. Resolving.